Source/Blazorise/Extensions/ParameterViewExtensions.cs

```diff
@@ -0,0 +1,34 @@
+﻿#region Using directives
+using System.Collections.Generic;
+using Microsoft.AspNetCore.Components;
+#endregion
+
+namespace Blazorise.Extensions;
+
+/// <summary>
+/// Helper methods for component parameters.
+/// </summary>
+public static class ParameterViewExtensions
+{
+    /// <summary>
+    /// Gets the value of the parameter with the specified name.
+    /// </summary>
+    /// <typeparam name="T">The type of the value.</typeparam>
+    /// <param name="parameters">Dictionary of all component paremeters.</param>
+    /// <param name="parameterName">The name of the parameter.</param>
+    /// <param name="currentValue">Last known parameter value.</param>
+    /// <param name="result">Receives the result, if any.</param>
+    public static void TryGetParameter<T>( this ParameterView parameters, string parameterName, T currentValue, out ComponentParameterInfo<T> result )
+    {
+        if ( parameters.TryGetValue<T>( parameterName, out var paramNewValue ) )
+        {
+            var changed = currentValue is IEnumerable<T> array && paramNewValue is IEnumerable<T> array2
```

Comment:
@David-Moreira see if this is a good way of checking for the array types and if it's fast enough.

Reply:
Yea I believe pattern matching is quite optimized that seems good. Also we do have this for a more generic implementation:
![image](https://user-images.githubusercontent.com/22283161/205520895-270e49b1-8008-4b8f-a1db-db924f21b11b.png)
![image](https://user-images.githubusercontent.com/22283161/205520919-91883768-7d60-40cb-ba08-46bbbba182fc.png)

But I believe you only want the `IEnumerable` there right?

Reply:
That seems like a reflection so it would be considerably slower. We just want to know that the value is `IEnumerable`.